abb_irb6650s_support/package.xml

```diff
@@ -34,10 +35,10 @@
   <buildtool_depend>catkin</buildtool_depend>
   <build_depend>roslaunch</build_depend>
```

Comment:
<build_depend>roslaunch</build_depend> should be replace   <depend>roslaunch</depend>

Reply:
replaced with `<test_depend>roslaunch</test_depend>`, as suggested by @gavanderhoorn